cs/ccxt/base/Throttler.cs

```diff
@@ -85,7 +85,7 @@ public async Task<Task> throttle(object cost2)
         if (!this.running)
         {
             this.running = true;
-            await this.loop();
+            Task.Run(() => { this.loop(); });
```

Comment:
ah wait, i was thinking of this directly:
```suggestion
            this.loop();
```
not?

Reply:
but actually idk, as you think, at least, await is removed and now it's already better. if you want leave it within task.run ✌️

Reply:
removed it let's see how it goes